include/exadg/incompressible_navier_stokes/user_interface/viscosity_model_data.h

```diff
@@ -0,0 +1,95 @@
+/*  ______________________________________________________________________
+ *
+ *  ExaDG - High-Order Discontinuous Galerkin for the Exa-Scale
+ *
+ *  Copyright (C) 2021 by the ExaDG authors
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *  ______________________________________________________________________
+ */
+
+#ifndef INCLUDE_EXADG_INCOMPRESSIBLE_NAVIER_STOKES_USER_INTERFACE_VISCOSITY_MODEL_DATA_H_
+#define INCLUDE_EXADG_INCOMPRESSIBLE_NAVIER_STOKES_USER_INTERFACE_VISCOSITY_MODEL_DATA_H_
+
+namespace ExaDG
+{
+namespace IncNS
+{
+/*
+ *  Turbulence model data.
+ */
+struct TurbulenceModelData
+{
+  TurbulenceModelData()
+    : turbulence_model(TurbulenceEddyViscosityModel::Undefined), constant(0.0), viscosity(0.0)
+  {
+  }
+
+  TurbulenceEddyViscosityModel turbulence_model;
+
+  // model constant and the fluid's kinematic (physical) viscosity
+  double constant;
+  double viscosity;
```

Comment:
I think we should avoid this redundancy of parameters. See comment https://github.com/exadg/exadg/pull/381/files#r1144851723